src/main/kotlin/racingcar/core/Racing.kt

```diff
@@ -0,0 +1,33 @@
+package racingcar.core
+
+import racingcar.ui.MessageCode
+import java.lang.IllegalArgumentException
+
+object Racing {
+
+    private const val INIT_TRY_NUMBER = 1
+    private const val INIT_CAR_NUMBER = 1
+    private const val SEPARATOR = " "
+
+    val cars = mutableListOf<Car>()
+
+    fun setCarInit() {
+        cars.clear()
+    }
+
+    fun setCars(carNumber: Int) {
+        if (carNumber < INIT_CAR_NUMBER) throw IllegalArgumentException(MessageCode.CarNumberException.message)
+        (INIT_CAR_NUMBER..carNumber).map {
+            cars.add(Car())
+        }
+    }
```

Comment:
`map`에서 반환되는 값을 이용하고 있지 않네요. 이럴경우 `forEach`를 사용해보는건 어떨까요?

아니면 이렇게 map 반환 값을 사용하도록 변경해보는것도 좋을 것 같아요 :)
```suggestion
    fun setCars(carNumber: Int): List<Car> {
        if (carNumber < INIT_CAR_NUMBER) throw IllegalArgumentException(MessageCode.CarNumberException.message)
       return (INIT_CAR_NUMBER..carNumber).map {
            Car()
        }
    }
```